### DISCUSSION TITLE
Bật phần mềm lên nhưng nó ở dạng cực tiểu

### DISCUSSION THREAD

**Discussion by ZeusFate** [d2phap/ImageGlass#29] [General] [answered]
Làm thế nào để phần mềm ở dạng maximum khi mở file ảnh. Tiếp nữa là ảnh luôn fullscreen khi nào ta kéo chuột lên trên thì nó mới hiện ra thanh công cụ

**Comment by d2phap** [comment, marked as answer]
bạn chỉ cần phóng to cửa sổ rồi tắt phần mềm, lần khởi động sau nó sẽ tự nhớ trạng thái của cửa sổ.

Còn về ảnh luôn fullscreen thì bạn phải chọn chức năng này mới được, bạn cũng có thể ẩn thanh công cụ đi (Ctrl + F1)
